### PR TITLE
[Execution][Benchmark] Add support for a pre-generating all transactions in the block-stm benchmark

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

Add a change to support pre-generating all transactions - this helps to isolate the noise from CPU usage in generating the transactions and also measures CPU utilization of execution only phase.

### Test Plan

```
cargo run --release -p aptos-transaction-benchmarks execute --generate-then-execute --num-accounts 100000 --block-size 50000 --concurrency-level-per-shard 10
```

### CODE REVIEW

File: aptos-move/aptos-transaction-benchmarks/src/transaction_bench_state.rs [hunk 1]
```diff
@@ -0,0 +1,326 @@
+// Copyright © Aptos Foundation
+
+use crate::{transactions, transactions::RAYON_EXEC_POOL};
```
Comment (sitalkedia) [inline aptos-move/aptos-transaction-benchmarks/src/transaction_bench_state.rs:3]: This is just moving the `TransactionBenchState` to a new module, no logic change.

**Review by danielxiangzl — APPROVED** [review]
LGTM!

**Review by grao1991 — APPROVED** [review]

File: aptos-move/aptos-transaction-benchmarks/src/main.rs [hunk 2]
```diff
@@ -181,8 +186,10 @@ fn execute(opt: ExecuteOpt) {
         opt.num_executor_shards,
         opt.concurrency_level_per_shard,
         opt.remote_executor_addresses,
-        opt.no_conflict_txns,
+        false,
+        //opt.no_conflict_txns,
```
Comment (manudhundi) [inline aptos-move/aptos-transaction-benchmarks/src/main.rs:190]: remove this commented line ?

**Review by manudhundi — APPROVED** [review]
Looks fine. Just left a minor comment

File: aptos-move/aptos-transaction-benchmarks/src/main.rs (hunk 2 above)
Comment (sitalkedia) [reply aptos-move/aptos-transaction-benchmarks/src/main.rs:190]: Good eye, I actually didn't mean to comment this out. Let me fix it.

**Pull request merged**